### DISCUSSION TITLE
lab submission

### DISCUSSION THREAD

**Discussion by morrisnjoka** [giswqs/geog-312#37] [Q&A] [answered]
hello, I am a student following your lectures online. Can I register and submit the lab assignment without paying? if yes, how?

**Comment by giswqs** [comment, marked as answer]
No. The university charges a fee for students enrolled in the course. All course materials and video recordings are freely available. You can follow them at your own pace. No need to enroll unless you want a certificate.

**Comment by rana-ruet** [comment]
Hi, I have completed the lab and I don't want to enroll the course. Is there any way I can have the solution of the lab?

**Reply by giswqs** [reply]
Sorry. Only enrolled students have access to the solutions posted on Canvas.
